### ISSUE TITLE
Firebase auth is working on Android device but not in Editor

### ISSUE THREAD

**Issue by Rainking80** [firebase/quickstart-unity#548]
### Please fill in the following fields:
Unity editor version: 
2019.2.16f1

Firebase Unity SDK version: 
firebase_unity_sdk_6.9.0

Firebase plugins in use (Auth, Database, etc.):
Auth, Functions, InstanceId

Additional SDKs you are using (Facebook, AdMob, etc.): 
just GooglePlayGames

Platform you are using the Unity editor on (Mac, Windows, or Linux): 
Windows

Platform you are targeting (iOS, Android, and/or desktop): 
Android (iOS support is set to active - to solve a different build error)

Scripting Runtime (Mono, and/or IL2CPP):
IL2CPP

### Please describe the issue here:
My build is working on my device (signing in via password/email or Google PlayGames) but I see the following error when I try to sign in inside the Unity Editor:
```
ApplicationException: app
Firebase.Auth.FirebaseAuth.GetAuthInternal (Firebase.FirebaseApp app, Firebase.InitResult& init_result_out) (at Z:/tmp/tmp.iBHfcxehsa/firebase/auth/client/unity/proxy/FirebaseAuth.cs:712)
Firebase.Auth.FirebaseAuth+<GetAuth>c__AnonStorey2.<>m__0 () (at Z:/tmp/tmp.iBHfcxehsa/firebase/auth/client/unity/proxy/FirebaseAuth.cs:180)
Firebase.FirebaseApp.TranslateDllNotFoundException (System.Action closureToExecute) (at Z:/tmp/tmp.G7nHbBPBcF/firebase/app/client/unity/proxy/FirebaseApp.cs:83)
Firebase.Auth.FirebaseAuth.GetAuth (Firebase.FirebaseApp app) (at Z:/tmp/tmp.iBHfcxehsa/firebase/auth/client/unity/proxy/FirebaseAuth.cs:179)
NetworkManager.InitializeFirebase (System.Func`1[TResult] LoginFunction) (at Assets/Scripts/Network/NetworkManager.cs:174)
NetworkManager.<authGoogle>b__22_0 (System.Boolean success) (at Assets/Scripts/Network/NetworkManager.cs:140)
GooglePlayGames.PlayGamesPlatform+<>c__DisplayClass36_0.<Authenticate>b__0 (System.Boolean success, System.String msg) (at Assets/GooglePlayGames/ISocialPlatform/PlayGamesPlatform.cs:351)
GooglePlayGames.BasicApi.DummyClient.Authenticate (System.Action`2[T1,T2] callback, System.Boolean silent) (at Assets/GooglePlayGames/BasicApi/DummyClient.cs:51)
GooglePlayGames.PlayGamesPlatform.Authenticate (System.Action`2[T1,T2] callback, System.Boolean silent) (at Assets/GooglePlayGames/ISocialPlatform/PlayGamesPlatform.cs:382)
GooglePlayGames.PlayGamesPlatform.Authenticate (System.Action`1[T] callback, System.Boolean silent) (at Assets/GooglePlayGames/ISocialPlatform/PlayGamesPlatform.cs:351)
GooglePlayGames.PlayGamesPlatform.Authenticate (System.Action`1[T] callback) (at Assets/GooglePlayGames/ISocialPlatform/PlayGamesPlatform.cs:316)
GooglePlayGames.PlayGamesLocalUser.Authenticate (System.Action`1[T] callback) (at Assets/GooglePlayGames/ISocialPlatform/PlayGamesLocalUser.cs:50)
NetworkManager.authGoogle () (at Assets/Scripts/Network/NetworkManager.cs:126)
NetworkManager.Start () (at Assets/Scripts/Network/NetworkManager.cs:120)

```

Since it is working on the device I have no clue where to find the issue... Libraries etc are fine, Resolving them etc did not change the behaviour.

### Please answer the following, if applicable:

I created a completely new project, so no other libraries should interfere.

What's the issue repro rate? (eg 100%, 1/5 etc)
100%

**Comment by paulinon** [comment]
Hi @Rainking80 Could you provide a minimal reproducible example of your project? Thanks!

**Comment by Rainking80** [comment]
this will be hard because without all the firebase and google credentials it will not run, right? and the second problem: even with just Firebase and Google libraries, the project is already 1 GB big because of (folder il2cpp_android_arm64-v8a and il2cpp_android_armeabi-v7a) inside Library. Do you want it anyway, and if yes, how should I send you a zip with this size? is google drive a possibility?

**Comment by dwulive** [comment]
You never need to include the library folder.  I can be recreated by doing a build.

**Comment by paulinon** [comment]
Hi @Rainking80 you may send using Google Drive via shareable link. Thanks.

**Comment by a-maurice** [comment]
Hi @Rainking80 

If I am understanding your issue correctly, you are using Google Play Games Services as your authentication provider, and when you try to use that in the Unity editor, you get that error?  If that is the case, that is because Google Play Games services sign-in is only supported on Android, as noted here: https://firebase.google.com/docs/auth/unity/play-games.

However, it should be handling that error more gracefully if that is the case, so please let us know if that is indeed the case.

**Comment by Rainking80** [comment]
Hi @a-maurice 

Google Play Games Services is included yes, but I use a platform-dependent compilation switch and in case it runs in the editor I use email/password credentials for firebase and not Google. The very same build is working on Android phones (also without Play Games sign-in).

@paulinon - unfortunately I ran into a different issue. I need to fix this first before I can send you the project. Hopefully this evening.

**Comment by Rainking80** [comment]
Hi, so here is what happened: I still saw the error message. So I deleted the Library folder to export the project. I gave it a last try and rebuild the project... and now it works. The error message has vanished - I can log in via the unity editor. I guess I had some corrupt files in the Library folder.

Thank you very much! ( I guess without you I would never have found the issue, or it would have taken ages)
